<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>981</width>
     <height>591</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="Btn_Scan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>扫描读卡器</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>220</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Open_card">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>891</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>打开读卡器</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Rd_card">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>读取的卡号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_read_card">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>360</y>
       <width>751</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_card">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>卡号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>姓名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>电话:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_reg">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Card">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>671</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>671</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_phone">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>671</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_find">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_recharge">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>充值</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_scan_money">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>661</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>941</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_refresh">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>550</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新全部顾客会员卡信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_destroy">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>550</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>注销卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>951</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Tab3_refrash">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>480</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>更新信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_tab_delete_item">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>480</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>删除选中商品</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>127</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_shopid">
        <property name="text">
         <string>商品编号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_shopname">
        <property name="text">
         <string>商品名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_sellprice">
        <property name="text">
         <string>售卖价格:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_shopbrand">
        <property name="text">
         <string>商品品牌：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Btn_tab4_addshop">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>360</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>添加商品</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>391</width>
       <height>179</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_shopid"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_shopname"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sellprice"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_shopbrand"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>页</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>页</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
